quizapp/src/main/java/billennium/quizapp/controller/CandidateController.java

```diff
@@ -0,0 +1,33 @@
+package billennium.quizapp.controller;
+
+
+import billennium.quizapp.entity.Candidate;
+import billennium.quizapp.exception.CandidateException;
+import billennium.quizapp.repository.CandidateRepository;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.UUID;
+
+import static billennium.quizapp.controller.ControllerConstants.CANDIDATE;
+import static billennium.quizapp.controller.ControllerConstants.EMAIL_PARAM;
+
+
+@RestController
+@RequiredArgsConstructor
+public class CandidateController {
+
+    private final CandidateRepository candidateRepository;
+
+    @GetMapping(CANDIDATE + EMAIL_PARAM)
+    @ResponseStatus(HttpStatus.OK)
+    public UUID getUserId(@PathVariable("email") String email) {
```

Comment:
jeśli to zwróci nam UUID to może ciężko będzie nam to obsłużyć. Może trzeba zwracać po prostu Stringa z UUID. do sprawdzenia